packages/lib-utils/src/components/list-view/ListView.stories.tsx

```diff
@@ -15,10 +23,24 @@ const meta: ComponentMeta<typeof ListView> = {
 export default meta;
 
 const Template: ComponentStory<typeof ListView> = (args) => {
+  const [selected, setSelected] = React.useState<string[]>([]);
+
   return (
     <BrowserRouter>
       <div style={{ overflowY: 'auto' }}>
-        <ListView {...args} />
+        <ListView
+          {...args}
+          onSelect={(e, isRowSelected, selectedData) =>
+            isRowSelected
+              ? setSelected([
+                  ...new Set([...selected, ...selectedData.map((i) => i.name as string)]),
```

Comment:
The `i.name as string` force cast may indicate an issue with `selectedData` type, can we adjust it so that the force cast is not needed?

Reply:
@vojtechszocs rReplaced it with `String(...)` or do you have a better idea? Also fixed an issue in default pagination logic causing sorting being applied only to page data - not all data